### PR TITLE
Update browser list that support the extension

### PR DESCRIPTION
**Pull request by venimus**
close https://github.com/aeternity/superhero-ui/issues/297

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -11,7 +11,8 @@
         v-if="isSupportedBrowser"
         class="supportedbrowser--alert"
       >
-        Alert - unsupported browser
+        Your browser does not support the Superhero extension.
+        Tipping and commenting will be disabled.
```
Comment (mradkov) [inline src/App.vue:15]: Can't we just use deeplinks to the webwallet if their browser is not supported?

Comment (venimus) [reply src/App.vue:15]: It is possible actually, is there an API for that listed?

Comment (venimus) [reply src/App.vue:15]: It seems however much bigger change for that. Still this is not really related to the PR, as it is for the "supported" ones.

Comment (venimus) [reply src/App.vue:15]: If we embed the wallet into the website the need of deeplinks would be only for mobile browsers

Comment (mradkov) [reply src/App.vue:15]: https://github.com/aeternity/superhero-wallet/blob/932e1b6926699174bd1583f5247163ca09bef4f8/tests/pages/index.html

**Comment by mradkov** [comment]
@IvayloPapazov is this finalized?

**Comment by venimus** [comment]
> @venimus is this finalized?

Yes, IMHO

**Pull request merged**